R/setup.R

```diff
@@ -72,6 +72,10 @@ shap_enable <- as.logical(Sys.getenv(
   "SHAP_ENABLE_OVERRIDE",
   unset = get(params_obj_name)$toggle$shap_enable
 ))
+comp_enable <- as.logical(Sys.getenv(
+  "COMP_ENABLE_OVERRIDE",
+  unset = get(params_obj_name)$toggle$comp_enable
+))
```

Comment:
**issue (blocking):** We should add this to `05-finalize.R` as well, right beside the other toggles saved to metadata: https://github.com/ccao-data/model-res-avm/blob/7eb26077c9917807208ddd19cd36c9ccdc0f75cc/pipeline/05-finalize.R#L90